pipeline-scripts/buildlib.groovy

```diff
@@ -115,13 +115,14 @@ def initialize_openshift_ansible() {
     echo "Initialized env.OPENSHIFT_ANSIBLE_DIR: ${env.OPENSHIFT_ANSIBLE_DIR}"
 }
 
-def initialize_openshift_jenkins() {
+def initialize_openshift_jenkins(branch) {
```

Comment:
Looks like "branch" is dead code.

Reply:
Arg... yup. Swore I removed that. It's definitely gone now.